src/apps/backend/modules/access-token/types.ts

```diff
@@ -12,8 +13,10 @@ export type AccessTokenPayload = {
 };
 
 export type CreateAccessTokenParams = {
-  password: string;
-  username: string;
+  contactNumber?: ContactNumber;
```

Comment:
Rename `ContactNumber` to `PhoneNumber`, similarly variable name, here and other places as well